docs/components/modeler/web-modeler/token-simulation.md

```diff
@@ -1,13 +1,15 @@
 ---
 id: token-simulation
 title: Token simulation
-description: You can use the token simulation feature to observe how the process will behave when it is executed.
+description: Use token simulation to get a deeper understanding of the BPMN execution semantics by playing, pausing, and resetting your simulation.
 ---
 
-You can use the token simulation feature to observe how the process will behave when it is executed.
+Use token simulation to get a deeper understanding of the BPMN execution semantics by playing, pausing, and resetting your simulation.
 
 ### Turn on/off
 
+First, open your BPMN diagram in Modeler.
```

Comment:
Token simuation is only available in the Design mode

Reply:
There may be other things broken in this document. I'm not touching the document up completely, but rather focus on clearly scoping the purpose.

Let's follow-up with everything else in a dedicated PR.

Reply:
Addressed via https://github.com/camunda/camunda-docs/pull/3011/commits/e7989c7e710ef136b7985561a8fc16eb99a9d7ba, now clearly stating that it is in Design mode.